content/quiz_events.md

```diff
@@ -4,21 +4,134 @@
 
 ### Challenges
+Challenges exist so quizzers can contest rulings made by quiz officials.
+
+The team captain or co-captain may challenge if:
+- There is reason to believe the quiz officials have applied the rule book incorrectly
+- There has been a mispronunciation by the quizmaster
+- There has been an invalid question
+- The next question has not been called
+    - A challenge must be made at the time of the quizmaster's decision and before the next question is called. If a time-out is called immediately after a ruling, and a quizzer stands to challenge, the challenge may be done prior to the time-out.
+- No conferring has occurred between the quizzer challenging and anyone else. Conferring with anyone else prior to a challenge will result in:
+    - The challenge being disallowed
+    - A foul being awarded to the quizzer.
+
+Once a team has requested an explanation of a ruling, and that explanation has been given, all teams lose the right to challenge.
+- If an explanation of a ruling is requested, the quizmaster should pause before giving the explanation, to allow all teams the opportunity to challenge.
+
+The result of a challenge may not be challenged.
+
+If the challenge is accepted, points or error points that were awarded for the initial ruling of the judge will be deducted.  Points or error points which reflect the new ruling of the judge will be recorded.  A new question of the same number will be given if the question is ruled illegal.
+
+When a challenge is made, the quizmaster must give each team’s captain or co-captain an opportunity to speak to the challenge before making a ruling on the challenge.  Whichever team is being challenged against will have the right to speak last.
 
 ### Protests
+Coaches, inclding assistant coaches, of the teams participating in a quiz may place the quiz under protest under the following conditions:
+- When the coach feels a logistical rule has been broken by the quiz officials. 
+    - Any rules regarding judgment calls on questions must be challenged by a captain/co- captain before they may be protested. 
+    - An example of a logistical rule being broken is when the quizmaster did not allow the full 30 seconds for the quizzer to answer when the quizzer was in context and had not given any wrong information.
+- The protest must be placed before the next question is called.
+    - This means the protest must apply to the question and ruling that just happened.
+- The coach may only confer with their assistant coach and substitute quizzer, before protesting.
+- The quiz officials and coaches will leave the room and attempt to come to an agreement.  If no agreement is reached within 10 minutes, the meet director will arbitrate and determine the just solution.
+- If a protest is launched after question number 20 is completed it must be done immediately after the ruling is announced.
+
+Broader, logistical issues do not require a Protest. Examples include:
```

Comment:
I added this section and would love thoughts.

Reply:
I'm not sure the section is necessary. From what I've observed, coaches and quizzers instinctively understand this.

Reply:
I heartily disagree. 

We are tightening the current rulebook. As such, it explicitly allows certain things and disallows certain things. And based off the context of the rules, I think it's clear that if we are explicitly allowing things, everything else is not allowed. 

Under Challenges and Protests, we lay out all the requirements for one to start/occur.

I think that implies that these are the only way "disagreements with the officials" can occur.

So I want to explicitly call out and ALLOW the "not really related to a specific event or specific ruling" stuff that we WANT quizzers/coaches bringing to the attention of the officials.

And you saying "instinctively understand" is correct, but has aspects of tribalism---I think newer coaches would hesitate to do something that isn't "stated as allowed"

Yes, I agree the second use of "logistical" is confusing. I'd love help making it less so. Basically it's "stuff that doesn't specifically apply to the question that just happened"

Reply:
I'm with Scott on this one. I'm staunchly anti-tribal. I'd soften the "meet outside" to "meet privately", but otherwise, I'm cool with the language here.

Reply:
I'm not strongly _against_ this rule and I could be persuaded otherwise... it's not strictly necessary but it takes nothing away either. I just think most of the section needs a complete rewrite if we want to include it. There need to be carefully distinguished definitions for all these three things:

1. logistical issue that could merit a challenge-less protest
2. logistical issue that can be informally raised outside of a protest
3. a judgment call on a question, which must be challenged before being protested

Part of the problem is that the rulebook as written ALREADY has confusing language. The example of the quizzer not being given 30 seconds when they were "in context and had not given incorrect information" is actually in my opinion something that SHOULD require a challenge first before a protest. If a quizzer was counted wrong when they should have been allowed to keep answering, that's 100% a "judgment call" related issue, NOT a "logistical" issue. Given this issue, I really don't think we should be complicating this rule further at this stage. It needs a rewrite.

Reply:
Should we be creating the definitions for 1 and 3 (the things talked about in the current rule book) in v1?

I can get on board with, only defining 2 and involving language regarding 2, IF definitions for 1 and 3 are defined.

Reply:
I'm of the opinion that we clean up the language but do little else for v1. You could add some notes, like what Jeremy has above, into the best practices doc, so we have something to start on for v-next. But I think anything beyond cleaning up language to remove confusion or ambiguity for v1 takes us out of scope for FE.

As to Scott's point: I think in v-next we should be creating expanded definitions of what Jeremy wrote. Probably most of it should end up in the rule book versus best practices, but we'll see when we get to it.

Reply:
I am removing my additions, tabling until v-next. 1 and 3 will remain undefined in v1.